app/templates/data_summary.html

```diff
@@ -32,7 +35,7 @@ <h1>Data Summary page</h1>
         </table>
```

Comment:
not sure why but this table with the items doesn't seem to be populating properly when i run your code on my end

Reply:
@JGSPS would you be able to try trouble shooting and find out why this is?

Reply:
![Screen Shot 2022-07-21 at 10 57 54 AM](https://user-images.githubusercontent.com/68709672/180246118-83ff73df-6dfa-4e0f-95c7-c1f4e1bb7d3c.png)
This is what I see, even though I've selected "Corn" and "Arabica Coffee" from the dropdown on the config page

Reply:
fixed